subsys/spm/secure_services.c

```diff
@@ -47,6 +53,42 @@ int spm_secure_services_init(void)
 }
 
 
+#ifdef CONFIG_SPM_SERVICE_READ
+typedef struct {
```

Comment:
Avoid typedefs; I am surprised checkpatch is passing?

Reply:
Removed.